### PR TITLE
Add Belvo

### PR DESCRIPTION
**Pull request by manuel-belvo**
Add Belvo.com

### CODE REVIEW

**Comment by phallobst** [comment]
Currently this website has a Similarweb rank of over 200,000 - however, it's climbing rapidly. I'd suggest to re-vist this PR in a week or two and examine the progress then.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Apart from the 200K rank issue, there's no linked image or regions array. Please add that so that the PR can be merged when/if the rank issue gets fixed.

**Comment by manuel-belvo** [comment]
Thank you for the feedback, folks. I'll work on it.

File: img/b/belvo.com.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+</style>
+<g>
+	<path class="st0" d="M43.6,34.6c-6.5,0-12.8,1.3-18.7,3.9V8.8H0v108.5h24.9v-3.7c5.9,2.5,12.3,3.8,18.7,3.7
+		c24.9,0,43.2-17.7,43.2-41.4S69,34.6,43.6,34.6z M43.7,93.8C33.4,93.8,26,86.3,26,76.1s7.7-17.9,17.7-17.9
+		c10.5,0,17.9,7.5,17.9,17.8C61.7,86.3,54.1,93.8,43.7,93.8L43.7,93.8z"/>
+	<path class="st1" d="M110.1,83.5c10.5,0,17.9,7.5,17.9,17.8s-7.6,17.8-17.9,17.8s-17.7-7.5-17.7-17.7
+		C92.3,91.2,100,83.5,110.1,83.5z M110.1,83.5c10.5,0,17.9,7.5,17.9,17.8s-7.6,17.8-17.9,17.8s-17.7-7.5-17.7-17.7
+		C92.3,91.2,100,83.5,110.1,83.5z"/>
+</g>
+</svg>
```
Comment (Carlgo11) [inline img/b/belvo.com.svg:17]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 110.3"><path fill="#00101e" d="M43.6 25.8a46 46 0 0 0-18.7 3.9V0H0v108.5h24.9v-3.7a46.2 46.2 0 0 0 18.7 3.7c24.9 0 43.2-17.7 43.2-41.4S69 25.8 43.6 25.8Zm.1 59.2A17.1 17.1 0 0 1 26 67.3c0-10.2 7.7-17.9 17.7-17.9 10.5 0 17.9 7.5 17.9 17.8A17.2 17.2 0 0 1 43.7 85Z"/><circle cx="110" cy="92.3" r="18" fill="#f9c806"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
The rank is over 500K now - there's hardly any use in pursuing this PR, I guess.

**Pull request closed without merging**